micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/TimeWindowFixedBoundaryHistogram.java

```diff
@@ -60,11 +60,6 @@ public TimeWindowFixedBoundaryHistogram(Clock clock, DistributionStatisticConfig
         NavigableSet<Double> histogramBuckets = distributionStatisticConfig
             .getHistogramBuckets(supportsAggregablePercentiles);
 
-        Boolean percentileHistogram = distributionStatisticConfig.isPercentileHistogram();
-        if (percentileHistogram != null && percentileHistogram) {
-            histogramBuckets.addAll(PercentileHistogramBuckets.buckets(distributionStatisticConfig));
```

Comment:
An earlier version added `&& supportsAggregablePercentiles` to the if condition here, which makes this correct, but upon a closer look, this was unnecessary in the first place. `PercentileHistogramBuckets` are already conditionally added in the call immediately before this removed code: `DistributionStatisticConfig#getHistogramBuckets`.